tests/ludwig/features/test_feature_utils.py

```diff
@@ -5,6 +5,111 @@
 from ludwig.features import feature_utils
 
 
+@pytest.fixture
+def to_module() -> torch.nn.Module:
+    return torch.nn.Module()
+
+
+@pytest.fixture
+def type_module() -> torch.nn.Module:
```

Comment:
@dennisrall This fixture and the `to_module()` one appear the same, and just instantiate the `PyTorch` `Module()` class.  Without a docstring, it is a bit difficult to justify it.  Thank you.

Reply:
Thanks for your feedback. I added a docstring for both fixtures and also for the `hash` method of the `LudwigFeatureDict`